gdal_subsetter/transform.py

```diff
@@ -1,55 +1,72 @@
-""" CLI for adapting a Harmony operation to GDAL. """
-from datetime import datetime
+"""Adapter for the Harmony GDAL Adapter service."""
```

Comment:
Changes to imports:

* Formatting.
* Removing things that are now used in `reformat.py` instead.
* Importing the main function from `reformat.py`.

That first item is the bulk of the diff, but they are much more readable 😅